### PR TITLE
Add classifications counters for classrooms and students.

### PR DESCRIPTION
**Pull request by simoneduca**
Also:
- Add table for student list.
- Add FontAwesome CDN.
- Add Spinner component.
- Some styling and tidying up.

### CODE REVIEW

**Comment by simoneduca** [comment]
@eatyourgreens or @shaunanoordin this is ready for review :)

File: src/presentational/Classroom.jsx [hunk 1]
```diff
@@ -40,20 +40,29 @@ export default class Classroom extends Component {
...
+            <th>Name</th>
+            <th>Classifications</th>
+            <th>Action</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr key={i}>
+            <td>{attributes.zooniverse_display_name}</td>
+            <td>{attributes.classifications_count}</td>
+            <td className="btn btn-warning" role="button">Remove</td>
```
Comment (eatyourgreens) [inline src/presentational/Classroom.jsx]: Table cells aren't clickable or focusable elements, so better to use a `<button>` element inside the table cell here.

File: src/presentational/Classroom.jsx [hunk 2]
```diff
@@ -40,20 +40,29 @@ export default class Classroom extends Component {
           return studentIds.indexOf( itm.id ) > -1;
         })
         .map((itm) => {
-          return itm.attributes.zooniverse_display_name
+          return itm. attributes
```
Comment (shaunanoordin) [inline src/presentational/Classroom.jsx]: There's an unexpected space between `itm.` and `attributes`, but this is a minor formatting hiccup. Code works fine.

**Comment by shaunanoordin** [comment]
Overall, the code looks good to me! That said, Jim raises a good point re: <td>s and <button>s, so I'll wait until that's addressed before merging.
- Code passes sanity check.
- Classrooms Overview page has some nice visual pizazz to it.
- Students listing looks great.

Tested on Chrome49/FF45/Safari9 on MacbookPro+OSX

Note: There were some issues loading classes on FF, but they happened only a few times and I haven't had any success replicating them consistently, so that may just be an issue with me.

Status: **Pending** - happy to merge after [td/button](https://github.com/zooniverse/wildcam-gorongosa-education/pull/101#discussion_r56688260) issue is addressed.

**Comment by simoneduca** [comment]
@shaunanoordin I've included the comments, thanks.

**Comment by shaunanoordin** [comment]
Sweet! I also see that Groups are now called Assignments.

Status: **LGTM**
Action: **Merging**

**Pull request merged**